automatic/kubernetes-cli/kubernetes-cli.nuspec

```diff
@@ -3,7 +3,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>kubernetes-cli</id>
-    <version>1.24.1</version>
```

Comment:
No need to include the changes in this file, as AU will update it when it updates the package.

Reply:
I've dropped this files changes. Pleas verify.